pages/draft-06/json-hyper-schema-release-notes.md

```diff
@@ -1,107 +1,104 @@
+`"hrefSchema"` replaces the `"method": "get"` use, but leverages URI Template variables so that client data-driven dynamic URIs are no longer limited to the query string. `"encType"` is no longer needed with this approach.
 
 `"submissionSchema"` directly replaces the `"method": "post"` use, with `"submissionEncType"` replacing `"encType"`.
 
-#### Removing `"method"`
+##### Removing `"method"`
 
-Draft-05 tried to restore the draft-03 behavior of `"method"`, but feedback told us that users found the change very confusing.  With `"schema"` split, the draft-05 behavior of `"method"` was no longer needed.
+Draft-05 tried to restore the draft-03 behavior of `"method"`, but feedback told us that users found the change very confusing. With `"schema"` split, the draft-05 behavior of `"method"` was no longer needed.
 
-We could have switched by to draft-04's `"method"` behavior, but in addition to producing more confusion from all of the back and forth, the draft-04 approach to `"method"` was not consistent with the rest of the LDO design anyway.  Most notably, it caused problems with the usage of `"targetSchema"` as described above.
+We could have switched by to draft-04's `"method"` behavior, but in addition to producing more confusion from all of the back and forth, the draft-04 approach to `"method"` was not consistent with the rest of the LDO design anyway. Most notably, it caused problems with the usage of `"targetSchema"` as described above.
 
-### Q: So how do I indicate which HTTP methods are supported on a link?
+#### Q: So how do I indicate which HTTP methods are supported on a link?
 
-A: Ideally, this is implicitly conveyed by your link relation type, which is the primary indicator of semantics across machine-oriented hypermedia in general.  [RFC 5988](https://tools.ietf.org/html/rfc5988) provides guidance on creating custom (a.k.a. "extension") link relations.
+A: Ideally, this is implicitly conveyed by your link relation type, which is the primary indicator of semantics across machine-oriented hypermedia in general. [RFC 5988](https://tools.ietf.org/html/rfc5988) provides guidance on creating custom (a.k.a. "extension") link relations.
 
 Several URI schemes and namespaces, such as the [UUID namespace in the `urn:` scheme](https://tools.ietf.org/html/rfc4122), or the [`tag:` scheme](https://tools.ietf.org/html/rfc4151), are particularly suitable for creating unique identifiers.
 
 And of course, there are ways to detect this at runtime such as HTTP's `"Allow"` response header, or attempting a method and handling a `405 Method Not Allowed` error accordingly.
 
-### Q: No, really. How do I _explicitly_ indicate which HTTP methods are supported on a link?
+#### Q: No, really. How do I _explicitly_ indicate which HTTP methods are supported on a link?
```

Comment:
suggestion: Change the format of this question. The way it is presented sounds like we were kidding with the reader in the previous answer. One possibility could be to remove the question and add the answer to the previous question by saying something like this: 

"If you need to _explicitly_ indicate which HTTP methods are supported, you can use 'targetHints' from draft-07 as an extension to draft-06..."

Reply:
I adopted the latter suggestion. Thank you.